selenium_tests/cross_listing/add_cross_listing_tests.py

```diff
@@ -0,0 +1,55 @@
+from ddt import ddt, data, unpack
+
+from selenium_common.base_test_case import get_xl_data
+
+from selenium_tests.cross_listing.cross_listing_base_test_case import \
+    CrossListingBaseTestCase, TEST_DATA_CROSS_LISTING_MAPPINGS
+
+
+@ddt
+class AddCrossListingTests(CrossListingBaseTestCase):
+
+    @data(*get_xl_data(TEST_DATA_CROSS_LISTING_MAPPINGS))
+    @unpack
+    def test_add_cross_listing_pairing(self, test_case,
+                                       primary_cid,
+                                       secondary_cid,
+                                       expected_result,
+                                       expected_text):
+        """
+        TLT-2589, AC #1 and 7
```

Comment:
Good point, and fixed.  I am opting to put in the original requirement Jira #, related selenium subtask Jira #, and the ACs (which doesn't get changed) the test cover.  Probably won't be linking to specific test case # since that goes through quite a bit of changes.